src/lib.rs

```diff
@@ -567,11 +574,22 @@ impl<'a> Exporter<'a> {
             CowStr::from(""),
         );
 
-        vec![
-            Event::Start(link_tag.clone()),
-            Event::Text(CowStr::from(reference.display())),
-            Event::End(link_tag.clone()),
-        ]
+        eprintln!("reference display: {}, file: {}", reference.display(), reference.file);
+
+        if !self.keep_wiki_links {
+            vec![
+                Event::Start(link_tag.clone()),
+                Event::Text(CowStr::from(reference.display())),
+                Event::End(link_tag.clone()),
+            ]
+        }
+        else {
+            vec![
+                Event::Text(CowStr::from("[[")),
+                Event::Text(CowStr::from(reference.display())),
+                Event::Text(CowStr::from("]]")),
+            ]
```

Comment:
This is the crux of the change. If we want our wikilinks to be preserved, then use text nodes for the output. 

Unfortunately there is a bug here that I'm stuck on. The resulting markdown prepends a backslash `\` to the link. Instead of `[[My Link to Another Concept]]` I'm getting `\[[My Link to Another Concept]]`.

The bug is reproduced in the test case `test_keep_wiki_links`